GuysNight.LethalCompanyMod.BalancedItems/Models/ItemPropertyOverride.cs

```diff
@@ -1,32 +1,106 @@
-﻿using GuysNight.LethalCompanyMod.BalancedItems.Utilities;
+﻿using System;
+using GuysNight.LethalCompanyMod.BalancedItems.Utilities;
 
 namespace GuysNight.LethalCompanyMod.BalancedItems.Models {
 	/// <summary>
 	/// Contains the data required for overriding various aspects of an item.
 	/// </summary>
 	public sealed class ItemPropertyOverride {
+		private ushort _averageValue;
```

Comment:
Should this be nullable ?

Reply:
Yes! But I wound up refactoring this to just be a property anyway. The value I was getting by having an explicit private field was useless given that this is a `ushort` and not a `short`.